### PR TITLE
[DT-2733] Ajax.js refactor

### PR DESCRIPTION
**Pull request by fboulnois**
### Addresses

https://broadworkbench.atlassian.net/browse/DT-2733

### Summary

Move `ajax.js` functionality into the appropriate files with Typescript, effectively deprecating `ajax.js` completely.

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/libs/ajax/ServiceStatus.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { getApiUrl, getOntologyUrl } from '../ajax'
+import { Config } from '../config'
```
Comment (rushtong) [inline src/libs/ajax/ServiceStatus.ts]: minor nit, it would be nice to clean up the import here.

File: src/libs/auth/auth.ts [hunk 2]
```diff
@@ -39,3 +39,10 @@ export const Auth = {
     await OidcBroker.signOut()
   },
 }
+
+// to log out user and redirect to home when response has 401 status
+// return responses with statuses in the 200s and reject the rest
```
Comment (rushtong) [inline src/libs/auth/auth.ts]: Minor nit: comments here look like they might be out of date since this method no longer:

> return responses with statuses in the 200s and reject the rest

I'm guessing that part was refactored out of this method some time ago.

**Review by rushtong — APPROVED** [review]
Nice cleanup, thank you. Address minor comments as you see fit 👍🏽

**Review by kevinmarete — APPROVED** [review]
Nice refactor!
